src/resmom/mom_main.c

```diff
@@ -7121,49 +6435,17 @@ static char	configusage[] = "%s -s insert scriptname inputfile\n"
 void
 usage(char *prog)
 {
-#ifdef	WIN32
-	fprintf(stderr,
-		"Usage: %s [-C chkdirectory][-d dir][-c configfile][-r|-p][-R port][-M port][-S port][-L log][-a alarm][-n nice]\n", prog);
-#else
-	fprintf(stderr,
-		"Usage: %s [-C chkdirectory][-d dir][-c configfile][-r|-p][-R port][-M port][-L log][-a alarm][-n nice]\n", prog);
-#endif
+	snprintf(log_buffer, sizeof(log_buffer), "%s",
```

Comment:
I think you should just a complete different implementation of usage for win32 and linux